PGIP/Output/Proof.hs

```diff
@@ -108,12 +108,17 @@ formatProofs format options proofs = case format of
         _ -> error $ "Failed reading the number " ++ show (todSec tod)
     }
 
-  convertTacticScript :: LP.ProofStatus G_proof_tree -> TacticScript
-  convertTacticScript ps =
-    let atp = read $ (\ (LP.TacticScript ts) -> ts) $ LP.tacticScript ps
-    in TacticScript { timeLimit = tsTimeLimit atp
-                    , extraOptions = tsExtraOpts atp
-                    }
+  convertTacticScript :: Maybe (LP.ProofStatus G_proof_tree)
+                      -> Maybe TacticScript
+  convertTacticScript Nothing = Nothing
+  convertTacticScript (Just ps) =
+    let ts = (\ (LP.TacticScript ts) -> ts) $ LP.tacticScript ps
+        atp = read ts
```

Comment:
maybe it is a better idea to use `Common.Uitls.readMaybe` instead of only testing the special case of an empty string.